inc/dropdown.class.php

```diff
@@ -1048,13 +1049,24 @@ static function getStandardDropdownItemTypes() {
 
 
    /**
-    * Display a menu to select a itemtype which open the search form
+    * Display a menu to select an itemtype which open the search form (by default)
     *
-    * @param $title     string   title to display
-    * @param $optgroup  array    (group of dropdown) of array (itemtype => localized name)
-    * @param $value     string   URL of selected current value (default '')
+    * @param string     $title     title to display
+    * @param array      $optgroup  (group of dropdown) of array (itemtype => localized name)
+    * @param string     $value     URL of selected current value (default '')
+    * @param array      $options
+    *
+    * @return void
    **/
-   static function showItemTypeMenu($title, $optgroup, $value = '') {
+   static function showItemTypeMenu(string $title, array $optgroup, string $value = '', array $options = []): void {
+      $params = [
+         'dropdownParentSelector' => null,
```

Comment:
I think it is preferable to use snake case naming for `$options`, as it is done almost everywhere in GLPI.
```suggestion
         'dropdown_parent_selector' => null,
```